generate_stats.py

```diff
@@ -32,13 +34,11 @@ def main():
 
     # get tags
     dagmc_tags = dagmc_stats.get_dagmc_tags(my_core)
-    
     # get Ranges of various entities
-    entity_types = [types.MBVERTEX, types.MBTRI]
-    entity_ranges = dagmc_stats.get_entity_ranges(my_core, all_meshset, entity_types)
-
-    report_stats(entity_ranges)
+    entity_types = [types.MBVERTEX, types.MBTRI, types.MBENTITYSET]
+    native_ranges = dagmc_stats.get_native_ranges(my_core, all_meshset, entity_types)
+    entityset_ranges = dagmc_stats.get_entityset_ranges(my_core, all_meshset, dagmc_tags['geom_dim'])
+    report_stats(native_ranges, entityset_ranges)
     
 if __name__ == "__main__":
-    main()
-    
+    main()
```

Comment:
I would suggest to put back a blank line before the end of file. 
It is not exactly a pep8 requirement I believe (pep8 only requires blank line around class and method definitions....)

Reply:
you can find docs on pep8 [[here](https://www.python.org/dev/peps/pep-0008/)]